Bio/KEGG/KGML/KGML_pathway.py

```diff
@@ -78,12 +78,15 @@ def __init__(self):
 
     def get_KGML(self):
         """Return the pathway as a string in prettified KGML format."""
-        header = "\n".join(['<?xml version="1.0"?>',
-                            "<!DOCTYPE pathway SYSTEM "
-                            '"http://www.genome.jp/kegg/xml/'
-                            'KGML_v0.7.1_.dtd">',
-                            "<!-- Created by KGML_Pathway.py %s -->" %
-                            time.asctime()])
+        header = "\n".join(
+            [
+                '<?xml version="1.0"?>',
+                "<!DOCTYPE pathway SYSTEM "
+                '"http://www.genome.jp/kegg/xml/'
+                'KGML_v0.7.1_.dtd">',
+                "<!-- Created by KGML_Pathway.py %s -->" % time.asctime(),
+            ]
+        )
```

Comment:
That might be clearer with literal ``\n`` characters in a multiline string, but that's not a black issues.